planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -1174,6 +1174,19 @@ FilteredByLanesObjects NormalLaneChange::filterObjectsByLanelets(
       continue;
     }
 
+    if (
+      check_optional_polygon(object, lanes_polygon.expanded_target) && is_lateral_far &&
+      is_before_terminal()) {
+      const auto ahead_of_ego =
+        utils::lane_change::is_ahead_of_ego(common_data_ptr_, current_lanes_ref_path, object);
+      if (object.kinematics.initial_twist_with_covariance.twist.linear.x < 1.0) {
```

Comment:
(minor) `1.0` can this be a parameter ? Or at least a named `constexpr` (e.g., `stopped_object_velocity_threshold`).

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8218/commits/60d6a97fad90c12a706406d354ce2b8ae5df1555

Reply:
I didn't make it a param because it make the PR unnecessary long. However, since there are many parts of the code has similar check (for example in the rss selection for parked and non parked), I will make a pr to refactor this.